comps/embeddings/langchain_multimodal/embedding_multimodal.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import datetime
+import os
+import time
+from typing import Union
+
+from dateparser.search import search_dates
+from embeddings_clip import vCLIP
+from langsmith import traceable
```

Comment:
We have already removed the langsmith from OPEA since it's not free for using. Please remove langsmith related code.

Reply:
updated